doc/man3/ENGINE_add.pod

```diff
@@ -585,9 +567,8 @@ L<OPENSSL_init_crypto(3)>, L<RSA_new_method(3)>, L<DSA_new(3)>, L<DH_new(3)>, L<
 
 =head1 HISTORY
 
-ENGINE_cleanup(), ENGINE_load_openssl(), ENGINE_load_dynamic(), and
-ENGINE_load_cryptodev() were deprecated in OpenSSL 1.1.0 by
-OPENSSL_init_crypto().
+ENGINE_cleanup() was deprecated in OpenSSL 1.1.0 by
+OPENSSL_init_crypto() and should not be used.
```

Comment:
@richsalz:
> Except that implies that the cleanup routine must be called, which is not true.  And it’s on the same manpage so I think it’s okay.

Well, the current text implies that `OPENSSL_init_crypto` should be called instead of `ENGINE_cleanup` ;-)